text/2019-11-11-mission.md

```diff
@@ -0,0 +1,72 @@
+# TiKV's Mission
```

Comment:
Maybe TiKV's Vision?

Reply:
Okay! :)

Reply:
I thought about it and referred to places like https://www.bain.com/insights/management-tools-mission-and-vision-statements/ https://www.bain.com/insights/management-tools-mission-and-vision-statements/

And I think Mission is still best. Do you still prefer vision?

Reply:
"Vision Statements are often combined to provide a statement of the company’s purposes, goals and values. "

Reply:
I think "mission" is not suitable for a project. For this, I think maybe "vision" or "goal" is better.

Reply:
Well, I'm not going to choose between two terms I think are incorrect 🤣 , so please choose the one you and our maintainers can agree on and I will alter the RFC.